Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -2294,37 +2310,40 @@ typedef SWIFT_ENUM_NAMED(NSInteger, RCStore, "Store", open) {
 };
 
 
-SWIFT_CLASS("_TtC10RevenueCat22StoreKitRequestFetcher")
-@interface StoreKitRequestFetcher : NSObject
-- (nonnull instancetype)init SWIFT_UNAVAILABLE;
-+ (nonnull instancetype)new SWIFT_UNAVAILABLE_MSG("-init is unavailable");
-@end
-
-
-
-@interface StoreKitRequestFetcher (SWIFT_EXTENSION(RevenueCat)) <SKRequestDelegate>
-- (void)requestDidFinish:(SKRequest * _Nonnull)request;
-- (void)request:(SKRequest * _Nonnull)request didFailWithError:(NSError * _Nonnull)error;
+SWIFT_CLASS("_TtC10RevenueCat16StoreKit1Wrapper")
+@interface StoreKit1Wrapper : NSObject <SKPaymentTransactionObserver>
+- (nonnull instancetype)init;
 @end
 
 
-SWIFT_CLASS("_TtC10RevenueCat15StoreKitWrapper")
-@interface StoreKitWrapper : NSObject <SKPaymentTransactionObserver>
-- (nonnull instancetype)init;
-@end
 
 @class SKPaymentQueue;
 @class SKPaymentTransaction;
 @class SKPayment;
 
-@interface StoreKitWrapper (SWIFT_EXTENSION(RevenueCat)) <SKPaymentQueueDelegate>
+@interface StoreKit1Wrapper (SWIFT_EXTENSION(RevenueCat)) <SKPaymentQueueDelegate>
```

Comment:
Wait how come this is exposed here if it's `internal`?  🧐

Reply:
(Not a regression or anything, I'm surprised this was always there).

Reply:
This works in API tests:
```objc
[StoreKit1Wrapper alloc];
```

Reply:
I can confirm this isn't isolated to this class, any `internal` class is being exposed outside for Objective-C. Bizarre.